### PR TITLE
Feature/151

### PR DESCRIPTION
**Pull request by MichaelXian**

### CODE REVIEW

File: database_config/init.sql [hunk 1]
```diff
@@ -109,7 +109,7 @@ CREATE TABLE public.job (
 );
 
 
-ALTER TABLE public.job OWNER TO admin;
+ALTER TABLE public.job OWNER TO postgres;
```
Comment (DConnorFong) [inline database_config/init.sql]: Why is this changed to postgres?

**Review by DConnorFong — CHANGES_REQUESTED** [review]

**Review by DConnorFong — APPROVED** [review]
Nice

File: server/controllers/searchController.ts [hunk 2]
```diff
@@ -128,9 +128,9 @@ async function getInnerQueryBuilder(search: SearchRequest, connection: Connectio
         .getRepository(Job)
         .createQueryBuilder("job")
         .select("job")
-        .addSelect(`setweight(to_tsvector(job.job_title), 'A') ||
```
Comment (cowmanjoe) [inline server/controllers/searchController.ts:131]: Little late but these are query literals here, they need to be single quotes.

File: server/controllers/searchController.ts [hunk 3]
```diff
@@ -156,17 +156,17 @@ function addSelects(search: SearchRequest, qb: SelectQueryBuilder<any>) {
 }
 
 function addWhere(search: SearchRequest, qb: SelectQueryBuilder<any>): void {
-    qb.where("job_document @@ to_tsquery(:keywords)", {keywords: `'${search.keywords}'`});
```
Comment (cowmanjoe) [inline server/controllers/searchController.ts:159]: Same here I believe.

**Review by cowmanjoe — COMMENTED** [review]
I'll make a ticket for fixing this. Can this merge be reverted? It breaks the server.

**Pull request merged**